src/Morphir/IR/SDK/Basics.elm

```diff
@@ -180,6 +180,90 @@ nativeFunctions =
                     ( Value.Literal _ (BoolLiteral v1), Value.Literal _ (BoolLiteral v2) ) ->
                         Ok (Value.Literal () (BoolLiteral (xor v1 v2)))
 
+                    _ ->
+                        Err (UnexpectedArguments [ arg1, arg2 ])
+            )
+      )
+    , ( "add"
+      , Native.binaryStrict
+            (\arg1 arg2 ->
+                case ( arg1, arg2 ) of
+                    ( Value.Literal _ (FloatLiteral v1), Value.Literal _ (FloatLiteral v2) ) ->
+                        Ok (Value.Literal () (FloatLiteral (v1 + v2)))
```

Comment:
Can you add cases to cover mixed combinations like `Float + Int` and `Int + Float`? You will need to use `toFloat` on `Int` before you can add them and they should both return `Float`.

This applies to `add`, `subtract`, `multiply` and `lessThan`.

Reply:
Test cases added.